packages/synapse-react-client/src/components/widgets/query-filter/JsonColumnFacetFilters.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Collapse } from '@mui/material'
+import React, { useState } from 'react'
+import { FacetFilterHeader } from './FacetFilterHeader'
+import { ColumnModel, FacetColumnResult } from '@sage-bionetworks/synapse-types'
+import { useQueryVisualizationContext } from '../../QueryVisualizationWrapper'
+import { useAtomValue } from 'jotai'
+import { tableQueryDataAtom } from '../../QueryWrapper/QueryWrapper'
+import { getCorrespondingColumnForFacet } from '../../../utils/functions/queryUtils'
+import { EnumFacetFilter } from './EnumFacetFilter/EnumFacetFilter'
+import { RangeFacetFilter } from './RangeFacetFilter'
+
+export type JsonColumnFacetFiltersProps = {
+  /* The parent ColumnModel that contains the JSON Subcolumns to show in this panel */
+  columnModel: ColumnModel
+  /* The set of JSON Subcolumn FacetColumnResults that will be shown in this panel */
+  facets: FacetColumnResult[]
+}
+
+/**
+ * Renders a set of facet filters that are all derived from the same JSON Column.
+ * @param props
+ * @constructor
+ */
+export default function JsonColumnFacetFilters(
```

Comment:
This component lets us logically group a set of JSON subcolumn facets that are all derived from the same column

Reply:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/2a388670-8659-496b-badd-722be78f748d)